pages/mods.vue

```diff
@@ -521,6 +561,34 @@ export default {
             formattedFacets.push(versionFacets)
           }
 
+          if (this.selectedEnvironments.length > 0) {
+            let environmentFacets = []
+            if (
```

Comment:
This whole thing can be simplified:
```js
let includesClient = this.selectedEnvironments.includes('client');
let includesServer = this.selectedEnvironments.includes('server');
if (includesClient && includesServer) {
    environmentFacets = [
        ['client_side:required'],
        ['server_side:required'],
    ]
} else {
   ...
}
```
This way it is more visually appealing and makes less searches

Reply:
Will do, thanks.